Sources/StreamChatUI/CommonViews/GradientView/GradientView.swift

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// A view that draws a gradient
+open class GradientView: _View {
+    /// A type representing gradient drawing options
+    public struct Content {
+        /// A type representing gradient direction
+        public enum Direction {
+            case vertical
+            case horizontal
+            case topRightBottomLeft
+            case topLeftBottomRight
+        }
```

Comment:
I think maybe we could use the struct pattern with static props that we have been using? 🤔 So that if the client wants, he can customize the Direction of the Gradient. Another possibility is just to add a new case `.custom(startPoint, endPoint)` WDYT?

Reply:
✅ Added